### PR TITLE
feature(API)#168283253]: admin can change user to mentor

### PR DESCRIPTION
**Pull request by mic7x4**
#### What does this PR do?
Have an API endpoint of  admin changes user to mentor.
#### Description of Task to be completed?
have the following endpoints working
POST /auth/signup
POST /auth/signin
PATH /v1/user/userid
#### How should this be manually tested?
after cloning the repo on your local machine cd in the directory and _run npm start_
You can test all those endpoints using postman
#### Any background context you want to provide?
Nope
#### What are the relevant pivotal tracker stories?
#168283253
#### Screenshots (if appropriate)
Screenshot from 2019-09-04 10-50-45

#### Questions:

### CODE REVIEW

File: server/routes/usersRoutes.js [hunk 1]
```diff
@@ -8,5 +8,7 @@ const usersRoute = express.Router();
 usersRoute.post('/auth/signup', UsersController.userSignUp);
 // User can signin
 usersRoute.post('/auth/signin', UsersController.userSignIn);
+// Change user to the Mentor
+usersRoute.patch('/user/:id', UsersController.userToMentor);
```
Comment (TheoOkafor) [inline server/routes/usersRoutes.js:12]: You need to also ensure that only an admin can change a user to a mentor. You can use a middleware to do this.

**Review by TheoOkafor — APPROVED** [review]
You can merge this, but remember my comment as you work.

File: server/routes/usersRoutes.js (hunk 1 above)
Comment (mic7x4) [reply server/routes/usersRoutes.js:12]: yeah I want to finish all the endpoints so that I can  add middleware and add authorization after

Comment (TheoOkafor) [reply server/routes/usersRoutes.js:12]: Alright

**Pull request merged**